src/fluxdiagrams.jl

```diff
@@ -42,11 +42,19 @@ function drawspc(spc::Species,path::String=".")
         end
     end
     if spc.adjlist != ""
-        mol = molecule.Molecule().from_adjacency_list(spc.adjlist)
```

Comment:
Can we do this one with Species the same way you do later?

Reply:
Yep! Done.